instrumentation/cassandra/cassandra-3.0/javaagent/src/test/java/CassandraClientTest.java

```diff
@@ -92,9 +90,9 @@ void syncTest(Parameter parameter) {
                           .hasKind(SpanKind.CLIENT)
                           .hasNoParent()
                           .hasAttributesSatisfyingExactly(
-                              equalTo(NET_SOCK_PEER_ADDR, "127.0.0.1"),
-                              equalTo(NET_SOCK_PEER_NAME, "localhost"),
-                              equalTo(NET_SOCK_PEER_PORT, cassandraPort),
+                              equalTo(NETWORK_TYPE, "ipv4"),
+                              equalTo(NetworkAttributes.NETWORK_PEER_ADDRESS, "127.0.0.1"),
+                              equalTo(NetworkAttributes.NETWORK_PEER_PORT, cassandraPort),
```

Comment:
This highlights an inconsistent use of static imports for this stuff. Not a deal breaker, but I think it would be nice to have the static imports to make more readable.

Reply:
I grok the above point about existing static imports making this even harder to get right. NBD.